bench/fake_k8s_api.py

```
@@ -0,0 +1,140 @@
+    },
+}
+
+NAMESPACE_OBJ = {
+    "kind": "Namespace",
+    "apiVersion": "v1",
+    "metadata": {"name": NAMESPACE},
+}
+
+
+class Handler(BaseHTTPRequestHandler):
+    def log_message(self, *args):
```

Ruff (0.15.7)

[warning] Missing type annotation for `*args`

(ANN002)